src/en/index.ts

```diff
@@ -773,14 +773,14 @@ const en = {
   pageDuplicateFailed: "Failed to duplicate page",
   copyOf: "Copy of",
   displayNameDescription:
-    "This could be your first name, or a nickname — however you'd like people to refer to you in Superthread.",
+    "This could be your first name, or a nickname. It will be displayed around the app.",
   embedVideo: "Embed Youtube video",
   removePullRequest: "Remove pull request",
   viewOnGitHub: "View on GitHub",
   setUpGHIntegration: "Set up GitHub integration",
   integrateSuperthread: "Integrate Superthread into your favorite git client.",
   githubIntegradionDescripiton:
-    "Link Superthread cards to GitHub pull requests. Include a Superthread Card-ID (e.g ST-123) in your branch name/s or pull request title/s to automatically show the statuses of the PRs in the Card.",
+    "Link Superthread cards to GitHub pull requests. Include a Superthread card ID (e.g C-123) in your branch name or pull request title to automatically show the PR status in the card.",
```

Comment:
@ben-bourdin451 I think github integration will work only if the branch name includes ST-${cardId}. Will it work with C- ?

Reply:
yes it also should work with `c-` prefix

Reply:
Ah, I misunderstood this message, and thought it had been written before Superthread had settled on the "C" prefix for cards. Happy to change this back if it makes things more clear.